Cargo.toml

```diff
@@ -369,10 +373,16 @@ idna_adapter = "=1.1.0"
 # alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "7fab7ee" }
 # alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "7fab7ee" }
 
+## alloy-evm
+# alloy-evm = { git = "https://github.com/alloy-rs/evm.git", rev = "95f6a8a" }
+# alloy-op-evm = { git = "https://github.com/alloy-rs/evm.git", rev = "95f6a8a" }
+
 ## revm
-revm = { git = "https://github.com/bluealloy/revm.git", rev = "2401c2c3" }
-op-revm = { git = "https://github.com/bluealloy/revm.git", rev = "2401c2c3" }
+revm = { git = "https://github.com/bluealloy/revm.git", rev = "b5808253" }
+op-revm = { git = "https://github.com/bluealloy/revm.git", rev = "b5808253" }
```

Comment:
This currently pins to https://github.com/bluealloy/revm/pull/2512, not yet included in a 23.* release

Reply:
Ref: https://github.com/bluealloy/revm/pull/2527

Pending release